### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by bala-thenightskycoder** [Gan-Tu/cppGFS2.0#63]
It showed the below error when trying to build in Mac OS X.

ERROR: /private/var/tmp/_bazel_balachandhar/14943156afbb9d37410525ecd02ad6fd/external/com_google_protobuf/BUILD:975:21: in proto_lang_toolchain rule @com_google_protobuf//:cc_toolchain: '@com_google_protobuf//:cc_toolchain' does not have mandatory provider 'ProtoInfo'.
INFO: Repository boringssl instantiated at:
  /Users/balachandhar/Documents/cppGFS2.0/WORKSPACE:52:10: in <toplevel>
  /private/var/tmp/_bazel_balachandhar/14943156afbb9d37410525ecd02ad6fd/external/com_github_grpc_grpc/bazel/grpc_deps.bzl:130:21: in grpc_deps
Repository rule http_archive defined at:
  /private/var/tmp/_bazel_balachandhar/14943156afbb9d37410525ecd02ad6fd/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository remotejdk11_macos instantiated at:
  /DEFAULT.WORKSPACE.SUFFIX:102:6: in <toplevel>
  /private/var/tmp/_bazel_balachandhar/14943156afbb9d37410525ecd02ad6fd/external/bazel_tools/tools/build_defs/repo/utils.bzl:201:18: in maybe
Repository rule http_archive defined at:
  /private/var/tmp/_bazel_balachandhar/14943156afbb9d37410525ecd02ad6fd/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository remote_java_tools_darwin instantiated at:
  /DEFAULT.WORKSPACE.SUFFIX:219:6: in <toplevel>
  /private/var/tmp/_bazel_balachandhar/14943156afbb9d37410525ecd02ad6fd/external/bazel_tools/tools/build_defs/repo/utils.bzl:201:18: in maybe
Repository rule http_archive defined at:
  /private/var/tmp/_bazel_balachandhar/14943156afbb9d37410525ecd02ad6fd/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
ERROR: Analysis of target '//src/protos/grpc:cc_chunk_server_lease_service_grpc' failed; build aborted: Analysis of target '@com_google_protobuf//:cc_toolchain' failed
INFO: Elapsed time: 8.781s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (72 packages loaded, 2959 targets configured)

**Comment by ArdxWe** [comment]
```
$  bazel build --features=-supports_dynamic_linker :gfs_client_main
INFO: Repository zlib instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /home/ardxwe/.cache/bazel/_bazel_ardxwe/5c5c1944d4a22c43212e6f2537dd3ef5/external/bazel_tools/tools/build_defs/repo/http.bzl:336:16: in <toplevel>
WARNING: Download from https://zlib.net/zlib-1.2.11.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
ERROR: An error occurred during the fetch of repository 'zlib':
   java.io.IOException: Error downloading [https://zlib.net/zlib-1.2.11.tar.gz] to /home/ardxwe/.cache/bazel/_bazel_ardxwe/5c5c1944d4a22c43212e6f2537dd3ef5/external/zlib/zlib-1.2.11.tar.gz: GET returned 404 Not Found
INFO: Repository build_bazel_rules_swift instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /home/ardxwe/.cache/bazel/_bazel_ardxwe/5c5c1944d4a22c43212e6f2537dd3ef5/external/bazel_tools/tools/build_defs/repo/http.bzl:336:16: in <toplevel>
INFO: Repository parallel_hashmap instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /home/ardxwe/.cache/bazel/_bazel_ardxwe/5c5c1944d4a22c43212e6f2537dd3ef5/external/bazel_tools/tools/build_defs/repo/http.bzl:336:16: in <toplevel>
INFO: Repository com_google_glog instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /home/ardxwe/.cache/bazel/_bazel_ardxwe/5c5c1944d4a22c43212e6f2537dd3ef5/external/bazel_tools/tools/build_defs/repo/http.bzl:336:16: in <toplevel>
ERROR: /home/ardxwe/.cache/bazel/_bazel_ardxwe/5c5c1944d4a22c43212e6f2537dd3ef5/external/com_google_protobuf/BUILD:150:1: @com_google_protobuf//:protobuf depends on @zlib//:zlib in repository @zlib which failed to fetch. no such package '@zlib//': java.io.IOException: Error downloading [https://zlib.net/zlib-1.2.11.tar.gz] to /home/ardxwe/.cache/bazel/_bazel_ardxwe/5c5c1944d4a22c43212e6f2537dd3ef5/external/zlib/zlib-1.2.11.tar.gz: GET returned 404 Not Found
ERROR: Analysis of target '//:gfs_client_main' failed; build aborted: no such package '@zlib//': java.io.IOException: Error downloading [https://zlib.net/zlib-1.2.11.tar.gz] to /home/ardxwe/.cache/bazel/_bazel_ardxwe/5c5c1944d4a22c43212e6f2537dd3ef5/external/zlib/zlib-1.2.11.tar.gz: GET returned 404 Not Found
INFO: Elapsed time: 1.917s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
    currently loading: @com_github_grpc_grpc//
```

**Comment by Gan-Tu** [comment]
Fixed via #64
